packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -595,7 +596,8 @@ export default class Meeting extends StatelessWebexPlugin {
           mediaRequests
         );
       }),
-      video: new MediaRequestManager((mediaRequests) => {
+      // @ts-ignore - config coming from registerPlugin
+      video: new MediaRequestManager(this.config.degradationPreferences, (mediaRequests) => {
```

Comment:
BTW, I'm adding also a `screenShareVideo` MediaRequestManager in PR#2545 and that made me realize that the limit is being enforced separately for screen share and separately for video, so in theory you could have twice the limit of macroblocks requested from Homer, but I guess it's OK, we can ignore that for now

Reply:
👍

Reply:
👍